DPPP/Predict.cc

```diff
@@ -434,10 +434,7 @@ namespace DP3 {
 
       // Call ApplyCal step
       if (itsDoApplyCal) {
-        if(itsMeasuresMutex == nullptr)
-          itsApplyCalStep.process(itsTempBuffer);
-        else
-          itsApplyCalStep.process(itsTempBuffer, itsMeasuresMutex);
```

Comment:
I don't know why itsMeasuresMutex was used here; it seems that an existing threading mutex was reused for HDF5.

Reply:
I might have done that, not sure though. Happy to blame you as well. In that case it's despicable!